src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -498,6 +498,12 @@ private function registerHttpCache(ContainerBuilder $container, array $config, X
 
         $loader->load('http_cache_tags.xml');
 
+        // TODO: Decide on how you would like to configure whether to use Varnish or Symfony (or any other?)
+        if (0 == count($config['http_cache']['invalidation']['varnish_urls'])) {
```

Comment:
We may allow to use both (to avoid such ifs). If a `varnish_urls` is defined, it's definitely to use Varnish.

Reply:
So how would your preferred config look like? If `varnish_urls` then Varnish, else Symfony? What if we want to integrate another one?